examples/express/index.ts

```diff
@@ -34,12 +34,94 @@ app.use('/workflow', serve<{ message: string }>(async (context) => {
     console.log('step 2 input', result1, 'output', output)
     return output
   })
+}, {
+  verbose: true,
+  retries: 0
 }));
 
 app.post("/get-data", (req, res) => {
   res.send("hey there");
 });
 
+// here we are
+
+app.use(
+  "/test",
+  serveQstashWorkflow(async (context: Parameters<RouteFunction<unknown>>[0]) => {
+    await context.run("test", async () => {
+      // console.log("test");
+    });
+  }, {
+    useJSONContent: true
+  })
+);
```

Comment:
is this part intended to be left in?

Reply:
You are commenting on an outdated version of the code, these comments don't exist in the new version.